app/views/templates/general_content.html.haml

```diff
@@ -1,8 +1,17 @@
+- if node.short_summary
```

Comment:
It is, but some nodes may not have a defined short summary (i.e. SectionHome nodes).

With that in mind, would it still make sense for us to shift this into app layout?